README.developers.md

```diff
@@ -341,14 +341,131 @@ $ ./run_reg_test.py odin_reg_strong
 ```
 and should be used when making changes to Odin.
 
-## Unit Tests
+# Unit Tests
 
 VTR also has a limited set of unit tests, which can be run with:
+
 ```shell
 #From the VTR root directory
 $ make && make test
 ```
 
+This will run `test_vtrutil`, `test_vpr`, and `test_fasm`. `test_archfpga` is not run.
```

Comment:
I think this actually reveals a bug. `test_vtrutil`, `test_vpr`, and `test_fasm` all use `add_test` in their CMake files:
https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/f4bd1bee06df02bb4d1f810d78b0dc48ff270a0a/utils/fasm/CMakeLists.txt#L58-L62

For some reason `test_archfpga` does not add their unit tests. It creates an executable for testing but does not add it to the CMake testing infrastructure:
https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/f4bd1bee06df02bb4d1f810d78b0dc48ff270a0a/libs/libarchfpga/CMakeLists.txt#L67-L72

I think this test should be added so that it is monitored by CI.